spec.bs

```diff
@@ -2659,20 +2659,59 @@ algorithms to achieve the outcomes described in the above explanatory content.
   or <{fencedframe}>'s [=container policy=], when it is constructed.
 </div>
 
+<div algorithm>
+  Create a new algorithm, called <dfn export>Create a Permissions Policy for a fenced
+  navigable</dfn>.
+
+  Given null or an [=element=] (|container|), this algorithm returns a new Permissions Policy.
+
+  1. [=Assert=]: if not null, |container| is a [=fenced navigable container=].
```

Comment:
Do you think we can assert that `|container|` is never null? This algorithm seems to only have one caller which assures that the input container is non-null, right?